libs/core-ui/src/lib/util/ImageStatisticsUtils.ts

```diff
@@ -1,6 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { localization } from "@responsible-ai/localization";
+
+import {
+  ILabeledStatistic,
```

Comment:
I remember there was some circular dependency issue with moving ILabeledStatistic here like this - hopefully I'm wrong and the builds will just pass